pep/app/pods/read/index/route.ts

```diff
@@ -16,4 +20,10 @@ export default class ReadIndex extends Route {
 
         copySearchToController(appController);
     }
+
+    redirect(model: any, transition: Transition) {
+        if (transition.to.name === this.routeName && this.currentUser.lastViewedDocumentId) {
```

Comment:
verify this, but im 99% sure you don't need this `transition.to.name === this.routeName` check here, as the `redirect()` will only fire if the user's final destination route is indeed read/index (i.e. it shouldn't fire if you are transitioning to read/document.